CanHazFunny/CanHazFunny/IJokeService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+public interface IJokeService
+{
+    public string GetJoke();
```

Comment:
```suggestion
    string GetJoke();
```
Interface members are public by default.